### PR TITLE
Updated to latest version

### PR DESCRIPTION
**Pull request by anmol26s**
## Problem
- *New Update available*

## Solution
- *Updated to new version*
- *Added git to update the app*

### CODE REVIEW

**Comment by anmol26s** [comment]
!testme

**Comment by anmol26s** [comment]
!testme

File: README.md [hunk 1]
```diff
@@ -7,7 +7,19 @@
...
+
+Version: 21.02.11
+
+### Interesting links
+
+- [YunoHost project](https://yunohost.org)
+- [Zap website](https://zotlabs.com/zap/)
+- [Zap code on codeberg](https://codeberg.org/zot/zap)
+- [Zap addons on codeberg](https://codeberg.org/zot/zap-addons)
+
```
Comment (ericgaspar) [inline README.md:19]:
```suggestion

```
Duplicates of lines 70 to 75

**Review by ericgaspar — CHANGES_REQUESTED** [review]
You seem to no longer use `app.src` and `app_addons.src`. Do you still want to keep it?

**Comment by anmol26s** [comment]
> You seem to no longer use `app.src` and `app_addons.src`. Do you still want to keep it?

Yes they will be used for the users who have already installed app before this PR. I don't know a way to bring them to git. For new installs it will use git and update using git. Zap don't use version tagging. So it better to use git for new users giving them flexibility if they want to go ahead and pull the app repo themself. Updating the package on day to day basis is hard.

File: README.md [hunk 2]
```diff
@@ -1,13 +1,17 @@
 # ZAP for YunoHost
 
 
-[![Integration level](https://dash.yunohost.org/integration/zap.svg)](https://dash.yunohost.org/appci/app/zap) ![](https://ci-apps.yunohost.org/ci/badges/zap.status.svg) ![](https://ci-apps.yunohost.org/ci/badges/zap.maintain.svg)  
+[![Integration level](https://dash.yunohost.org/integration/zap.svg)](https://dash.yunohost.org/appci/app/zap) ![](https://ci-apps.yunohost.org/ci/badges/zap.status.svg) ![](https://ci-apps.yunohost.org/ci/badges/zap.maintain.svg)
```
Comment (ericgaspar) [inline README.md:4]:
```suggestion
[![Integration level](https://dash.yunohost.org/integration/zap.svg)](https://dash.yunohost.org/appci/app/zap) ![](https://ci-apps.yunohost.org/ci/badges/zap.status.svg) ![](https://ci-apps.yunohost.org/ci/badges/zap.maintain.svg)  
```

Comment (ericgaspar) [reply README.md:4]: You keep removing those two spaces ... don't, it acts like a line break in the Markdown.

**Review by ericgaspar — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (anmol26s) [reply README.md:4]: I will be very careful on this from now onward.

**Review by ericgaspar — APPROVED** [review]

**Pull request merged**